routers/web/repo/compare.go

```diff
@@ -845,7 +845,15 @@ func CompareDiff(ctx *context.Context) {
 		}
 	}
 
-	ctx.Data["IsProjectsEnabled"] = ctx.Repo.CanWrite(unit.TypeProjects)
+	if ctx.Repo.CanRead(unit.TypeProjects) {
```

Comment:
I think it means wether the creator can assign projects on this page. So should we change it from `CanWrite` -> `CanRead`?

Reply:
That would hide the project section from the create PR sidebar. I'm okay with it, but we might want something else if we are going to support auto-assigning issues to projects using the issue template at some point.